packages/renderer/src/components/exporters/google-sheets/EditSheetsExporter.tsx

```diff
@@ -74,13 +75,7 @@ const EditSheetsExporter: React.FC<EditSheetsExporterProps> = ({
   return (
     <div className={styles.container}>
       <Card className={styles.card}>
-        <Image
-          className={styles.logo}
-          src={exporterConfig.logo}
-          roundedCircle
-          width={100}
-          height={100}
-        />
+        <Image className={styles.logo} roundedCircle width={100} height={100} />
         <Card.Body className={styles.cardBody}>
```

Comment:
Why did you removed the `src`?

Reply:
which src?

Reply:
The component was:

```
<Image
          className={styles.logo}
          src={exporterConfig.logo}
          roundedCircle
          width={100}
          height={100}
        />
```

With a `src` attribute. It is an image component, without the `src` of the image, it is useless.

Reply:
there is no logo under exporterConfig so I guess it's not supposed to work. lint/ts showed this error:
`TS2339: Property 'logo' does not exist on type 'GoogleSheetsConfig'.`

Reply:
And when you run `yarn watch`, do you see any missing icon?

Reply:
icons are missing with and without my change